iphone/Classes/TiLogServer.m

```diff
@@ -223,6 +242,23 @@ +(void)startServer
 		connections = [[NSMutableArray alloc] init];
 	}
 
+	if (headers == nil) {
+		NSString* version = [NSString stringWithCString:TI_VERSION_STR encoding:NSUTF8StringEncoding];
+		NSDictionary* map = [[NSDictionary alloc] initWithObjectsAndKeys:
```

Comment:
maybe not yet. I like Hans to take a look too. by the way, I did the functional tests, it didn't work :( they are still happily logging individually, even though i set the same port number for both apps.